src/lib.rs

```diff
@@ -39,32 +39,33 @@ pub struct Finder {
 impl Finder {
     /// `from_pb` is used when you can use your own timezone data, as long as
     /// it's compatible with Proto's desc.
-    pub fn from_pb(tzs: gen::Timezones) -> Finder {
-        let mut f: Finder = Finder {
+    #[must_use]
```

Comment:
Can you explain reason to add this `#[must_use]`?

Reply:
It merely indicates to the compiler that the return value from the function should be assigned to _something_, even if that something isn't being used. It just ensures that if the return value is being thrown away, this is done consciously and not by accident. One can assign the return value to something a variable like `_throwaway` which the compiler will throw away, but at least it's done on purpose.

Ref: https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-must_use-attribute